canonical-websites/docs.jujucharms.com
# Make redirects more inclusive

This ensures that all old docs can redirect with any combination of the prefixes `/devel`, `/stable`, `/en`, or none.

Fixes https://github.com/canonical-web-and-design/docs.ubuntu.com/issues/214

QA
--

`./run`, and go to:

- http://0.0.0.0:8033/stable/getting-started
- http://0.0.0.0:8033/getting-started.html
- http://0.0.0.0:8033/en/getting-started
- http://0.0.0.0:8033/authors-charm-writing
- http://0.0.0.0:8033/developer-getting-started.html
- http://0.0.0.0:8033/en/developer-getting-started.html
- http://0.0.0.0:8033/devel/en/developer-getting-started.html
- http://0.0.0.0:8033/en/reference-install
- http://0.0.0.0:8033/stable/en/reference-install

Check in every case you're redirected to the right page.
